v4l2/v4l2_encode.cpp

```diff
@@ -58,6 +58,15 @@ bool V4l2Encoder::start()
     streamFormat.streamFormat = AVC_STREAM_FORMAT_ANNEXB;
     m_encoder->setParameters(VideoConfigTypeAVCStreamFormat, &streamFormat);
 
+    if (m_svct) {
+        VideoTemporalLayers layers;
+        layers.numLayersMinus1 = 1; // It is two layers actually
+        layers.bitRate[0] = m_videoParams.rcParams.bitRate / 2;
+        layers.bitRate[1] = m_videoParams.rcParams.bitRate;
```

Comment:
Hi @LindaYu17 , 
We can delete the line "layers.bitRate[1] = m_videoParams.rcParams.bitRate;", because the second layer is the highest layer in this case, and whose bitrate is assigned by "m_videoParams.rcParams.bitRate" in vaapiencoder_xxx.cpp file.